docs/api/flink/Function.md

```diff
@@ -695,6 +695,28 @@ SELECT ST_NumInteriorRings(ST_GeomFromText('POLYGON ((0 0, 0 5, 5 5, 5 0, 0 0),
 
 Output: `1`
 
+## ST_NumPoints
+
+Introduction: Returns number of points in a LineString.
+
+!!!note
+    If any other geometry is provided as an argument, an IllegalArgumentException is thrown. 
+    Example: 
+    `SELECT ST_NumPoints(ST_GeomFromWKT('MULTIPOINT ((0 0), (1 1), (0 1), (2 2))'))`
+
+    Output: `IllegalArgumentException: Unsupported geometry type: MultiPoint, only LineString geometry is supported.`
+
+Format: `ST_NumPoints(geom: geometry)`
+
+Since: `v1.4.1`
+
+Spark SQL example:
```

Comment:
This flink doc should not have `SparkSQL` in it

Reply:
Sorry about that, removed